### DISCUSSION TITLE
Milwaukee M18 charger

### DISCUSSION THREAD

**Discussion by MarkAlbe** [aderusha/DDD-Printable-Wall-Control-System#35] [Q&A]
Has anyone done a version of the tool hook for the M18\M12 charger that does both? 

If not any tips on what to use to modify whats there to fit that model. I think it just needs slightly more space between the mounts for the bottom of the charger. I have never actually modified models I found online or created my own but I am willing to do the work if someone can point me to a good tutorial.

Also, amazing resource here. I am just getting started using it. Good work to all of you who have contributed. 

Thanks

**Comment by aderusha** [comment]
I'm am 100% willing to make an M18 charger mount if anyone wants to buy me a Big Red Tool with an M18 battery and charger :D
